backend/app.py

```
@@ -0,0 +1,196 @@
+                    "description": item.payload.get("description", ""),
+                    "quantity": item.payload.get("quantity", 1),
+                    "location": item.payload.get("location", ""),
+                    "storage_box": item.payload.get("storage_box", ""),
+                    "brand": item.payload.get("brand", ""),
+                    "size": item.payload.get("size", ""),
+                    "condition": item.payload.get("condition", ""),
+                    "image_data": item.payload.get("image_data", "")
+                })
+        
+        print(f"DEBUG: Found {len(filtered_results)} similar items for user {search.username}")
+        return {"success": True, "results": filtered_results}
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)